css/main.css

```diff
@@ -131,16 +143,61 @@ textarea {
 
   #section-1 {
     display: flex;
+    width: 45%;
   }
-
+  
   .vertical-text {
     flex:1;
+    color: #999;
+  }
+
+  .vertical-text p {
+    transform: rotate(-90deg);
+    -webkit-transform: rotate(-90deg);
+    -moz-transform: rotate(-90deg);
+    text-align: center;
   }
   
   .full-image {
     flex:5;
   }
 
+  .full-image img {
+    height: 94vh;
+    width: 100%;
+  }
```

Comment:
BUG: Setting both the image height and width usually results This height is making the image look stretched.  I saw this suggested fix for this kind of problem: https://alligator.io/css/cropping-images-object-fit/. This fix on .full-image img {} will help.
```object-fit:cover;
  }
```